cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -958,13 +958,13 @@ function Install_Cni_Binaries {
   $release_url = "${env:WINDOWS_CNI_STORAGE_PATH}/${env:WINDOWS_CNI_VERSION}/"
   $tgz_url = ($release_url +
               "cni-plugins-windows-amd64-${env:WINDOWS_CNI_VERSION}.tgz")
-  $sha_url = ($tgz_url + ".sha1")
-  MustDownload-File -URLs $sha_url -OutFile $tmp_dir\cni-plugins.sha1
-  $sha1_val = ($(Get-Content $tmp_dir\cni-plugins.sha1) -split ' ',2)[0]
+  $sha_url = ($tgz_url + ".sha512")
+  MustDownload-File -URLs $sha_url -OutFile $tmp_dir\cni-plugins.sha512
```

Comment:
gsutil ls -l gs://gke-release/cni-plugins/v0.8.5-gke*

currently .sha512 is not available, only .sha1

Reply:
is there a script to generate the sha1 files? or manually did it? i'll generate sha512 files.

Reply:
sha1 files are there, sha512 is missing currently

Reply:
i meant how those sha1 files are created. via a script or a command? i don't have permission to `gsutil cp` to the bucket.

Reply:
From https://github.com/kubernetes/kubernetes/blob/e99dbbde62015c15b516cdf39edb8d32fcd71729/cluster/gce/config-common.sh#L143, the bucket is `k8s-artifacts-cni`.

`gsutil ls -l gs://k8s-artifacts-cni/release/v0.8.7/cni-plugins-windows*`

shows that the sha512 files are present. I'm not sure why the `gke-release` bucket is different, but for this PR this change looks good.

Reply:
replied in bug